plugins/addon/type.go

```diff
@@ -18,8 +20,20 @@ func ReplaceTypes(u *model.Universe) error {
 		return err
 	}
 
+	projectInfo, err := scaffold.LoadProjectFile("PROJECT")
```

Comment:
Plugins should not need to read the `PROJECT` file, they should receive all the required info when called.

Reply:
This is still unsolved.

Reply:
I think will be not easy to pass the info for the plugin in this scenario. In order to simplify, I think we can use/check the PROJET file. @mengqiy do you see any problem with? Can we move in this way or do you think that it is a problem/blocker for we merge this PR?

Reply:
Would you mind raising an Issue about this when this get merged? So that we consider it for future PR. Plugins will suffer changes so I don't think this should block it.

Reply:
I don't think plugins should be manually reading the project file -- they should be passed the desierialized form as input.

Reply:
I definetely agree.

However, being the plugins in such an early state, I feel that they will change a lot before being available so we may be able to leave that for a plugin related PR and leave it here in this unortodox way. Maybe a vomment saying that plugins shouldnt need to access that file, that the info should be serialized through parameters could be added here in order not to forget.

Reply:
Done the change requested.

Reply:
Closing since it was done.